upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -261,6 +266,36 @@ func (_ *Subnet) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Subnet) error {
 		}
 	}
 
+	if changes.DNS64 != nil {
+		request := &ec2.ModifySubnetAttributeInput{
+			SubnetId:    e.ID,
+			EnableDns64: &ec2.AttributeBooleanValue{Value: e.DNS64},
+		}
+		_, err := t.Cloud.EC2().ModifySubnetAttribute(request)
+		if err != nil {
+			return fmt.Errorf("error enabling DNS64 on subnet: %v", err)
+		}
+	}
+
+	if changes.ResourceBasedNaming != nil {
+		request := &ec2.ModifySubnetAttributeInput{
+			SubnetId:                                e.ID,
+			EnableResourceNameDnsAAAARecordOnLaunch: &ec2.AttributeBooleanValue{Value: e.ResourceBasedNaming},
+		}
+		_, err := t.Cloud.EC2().ModifySubnetAttribute(request)
+		if err != nil {
+			return fmt.Errorf("error enabling resource based dns on subnet: %v", err)
+		}
+		request = &ec2.ModifySubnetAttributeInput{
+			SubnetId:                             e.ID,
+			EnableResourceNameDnsARecordOnLaunch: &ec2.AttributeBooleanValue{Value: e.ResourceBasedNaming},
+		}
+		_, err = t.Cloud.EC2().ModifySubnetAttribute(request)
```

Comment:
Don't you need to set `PrivateDnsHostnameTypeOnLaunch`? 
Why two separate `ModifySubnetAttribute` calls? You can't set multiple attributes in one call?